core/src/test/java/org/elasticsearch/search/aggregations/pipeline/BucketScriptIT.java

```diff
@@ -238,6 +243,52 @@ public void testInlineScript2() {
         }
     }
 
+    public void testInlineScript3() {
+        SearchResponse response = client()
+            .prepareSearch("idx")
+            .addAggregation(
+                dateRange("range")
+                    .field(FIELD_5_NAME)
+                    .addRange("now-1d", "now")
```

Comment:
Ok! As what is really being tested here is the combination of 2 aggs, I suppose the date does not really matters. So I'll use a fixed date then.